spec/models/miq_worker/systemd_common_spec.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.describe MiqWorker::SystemdCommon do
+  describe ".service_base_name" do
+    before { MiqWorkerType.seed }
+
+    it "every worker has a matching systemd target and service file" do
+      expected_units = (Vmdb::Plugins.systemd_units + Rails.root.join("systemd").glob("*.*")).map(&:basename).map(&:to_s)
+
+      expected_units.delete("manageiq.target")
+
+      found_units = MiqWorkerType.worker_class_names.flat_map do |klass_name|
+        klass = klass_name.constantize
+        service_base_name = klass.service_base_name
+
+        service_file = "#{service_base_name}@.service"
+        target_file  = "#{service_base_name}.target"
+
+        [service_file, target_file]
+      end
+
+      expect(expected_units).to match_array(found_units)
```

Comment:
Just realized this should have been the other way around...

```
      expect(found_units).to match_array(expected_units)
```

but it's fine.

Reply:
@Fryguy I think this was actually correct originally.  I swapped this in the follow-up PR but now that I'm running one switching this around prints missing unit files as "extra" rather than "missing".